tests/mocks/account_retriever.go

```diff
@@ -8,9 +8,89 @@ import (
 	client "github.com/cosmos/cosmos-sdk/client"
 	types "github.com/cosmos/cosmos-sdk/types"
 	gomock "github.com/golang/mock/gomock"
+	crypto "github.com/tendermint/tendermint/crypto"
 	reflect "reflect"
 )
 
+// MockAccount is a mock of Account interface
+type MockAccount struct {
```

Comment:
Is this used somewhere?

Reply:
This is a mock file generated by MockGen. It like is used in module tests. See `make mocks`

Reply:
TIL. Thanks 👍